python/tests/test_fastquant.py

```diff
@@ -37,12 +37,19 @@ def test_get_stock_data():
     assert isinstance(stock_df, pd.DataFrame)
 
     # Test getting yahoo when (default) phisix fails on a non PSE SYMBOL
-    stock_df = get_stock_data(
-        YAHOO_SYMBOL, DATE_START, DATE_END
-    )
+    stock_df = get_stock_data(YAHOO_SYMBOL, DATE_START, DATE_END)
     assert isinstance(stock_df, pd.DataFrame)
 
 
 def test_get_crypto_data():
+    # try default crypto exchange, which is binance
     crypto_df = get_crypto_data(CRYPTO_SYMBOL, DATE_START, DATE_END)
     assert isinstance(crypto_df, pd.DataFrame)
+    # test other exchanges work
+    from fastquant.crypto import CRYPTO_EXCHANGES
+
+    for ex in CRYPTO_EXCHANGES:
```

Comment:
Let's loop through both `CRYPTO_EXCHANGES` and a new list `CRYPTO_SYMBOLS`, where each element is a valid symbol corresponding to each exchange listed in `CRYPTO_EXCHANGES`. This should fix the failing tests.